transcom/mymove
# INT B-18313 issue fixes and storybook updates

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18313)
## [I-2610](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A917833)
## [I-12602](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A917202)

## Summary

This PR includes fixes for two issues (listed above) as well as updating the storybook for the `Summary` component. 

PR does the following:
- fixes typo in customer helper header when they have more than one move
- requires customer to validate their profile when they have anything in `currentMove` instead of `previousMoves`
- updates logic of helper box based on `currentMove` length
- updates tests
- updates storybook for `Summary` component

### How to test

1. Access MM as a customer
2. Create a move, no need to set up shipments
3. Head back to the dashboard
4. When going back to the dashboard, make sure the correct helper box is showing (tells you to validate your profile)
5. Click `Create a Move` button
6. This should navigate you to the profile validation flow
